### ISSUE TITLE
Button Bar Not Displaying

### ISSUE THREAD

**Issue by KeironO** [bittobennichan/Flask-MDE#7]
The button bar is not displaying, instead I have:

```
 <div id="wmd-button-bar"></div> 
```

Being injected into the html as:

```
            &lt;div id=&#34;wmd-button-bar&#34;&gt;&lt;/div&gt;<textarea id="wmd-input" name="editor">
```

**Comment by bittobennichan** [comment]
@KeironO Can you give a Minimal, Reproducible Example?

**Comment by bittobennichan** [comment]
@KeironO Can you please give a Minimal, Reproducible Example?

**Comment by bittobennichan** [comment]
@KeironO This is now fixed in v1.2.1
